experiments/swdt/cmd/kubernetes.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"swdt/apis/config/v1alpha1"
+	"swdt/pkg/pwsh/executor"
+	"swdt/pkg/pwsh/kubernetes"
```

Comment:
This package seems missing to me

Reply:
lol what...

Reply:
Unless you prefer to work on it, shall I do it? I'd need some bearings on canonical K8s project CI or its just GHA workflow w/ Go linters and `go test` jobs?

Reply:
feel free to add it, we have some other projects using like https://github.com/kubernetes-sigs/windows-operational-readiness/tree/main/.github/workflows

Reply:
It looks like you've already added it a minute ago in https://github.com/kubernetes-sigs/sig-windows-dev-tools/commit/a4b06be65f5a6e146318ec6d73229b5b3e1de267 - not the right approach?

Reply:
@knabben By the way, what is the K8s sentiment to use [pre-commit ](https://pre-commit.com/) hooks? Is it in any way popular as a first line of attack w.r.t Go linting and testing?

Reply:
i do prefer to run this on CI, it can be fine if optional though.. have a linter configured on my IDE

Reply:
yeah, optional, definitely and as long as you don't run `pre-commit install`  in he SWDT repo, you don't suffer :)

Reply:
can you open a PR for it?

Reply:
yes, I am going to, not sure if tonight, but tomorrow for sure.